spec/features/admin/resources_integration_spec.rb

```diff
@@ -160,4 +163,49 @@
       end
     end
   end
+
+  context "with event that has filters defined" do
+    around do |example|
+      Event.class_eval do
+        def self.filters
```

Comment:
Please use a more alchemy related name for this class method. For instance `alchemy_resource_filters`